base_layer/wallet_ffi/src/lib.rs

```diff
@@ -6365,6 +6379,45 @@ pub unsafe extern "C" fn wallet_set_base_node_peer(
     }
     true
 }
+/// Gets all seed peers known by the wallet
+///
+/// ## Arguments
+/// `wallet` - The TariWallet pointer
+/// `error_out` - Pointer to an int which will be modified to an error code should one occur, may not be null. Functions
+/// as an out parameter.
+///
+/// ## Returns
+/// `TariPublicKeys` - Returns a list of all known public keys
+///
+/// # Safety
+/// None
+#[no_mangle]
+pub unsafe extern "C" fn wallet_get_seed_peers(wallet: *mut TariWallet, error_out: *mut c_int) -> *mut TariPublicKeys {
```

Comment:
What's the purpose of the function? So they can be listed as connectable base nodes?

Reply:
yeah